src/browser/views.py

```diff
@@ -306,6 +310,23 @@ def repo_delete(request, repo_base, repo):
 
 
 @login_required
+def repo_delete_collaborator(request, repo_base, repo_owner, repo_name):
+  '''
+  removes self as a collaborator from the specified repo (repo_name)
+  '''
+  
+  username = request.user.get_username()
+  manager = DataHubManager(user=username)
+  # TODO: Add in logic for this method in manager.py (currently, repo_delete_collaborator only removes
+  #       the collaborator from the Django Collaborators table)
+  # manager.repo_delete_collaborator(username, repo_name, repo_owner)
+  
+  user = User.objects.get(username=username)
+  Collaborator.objects.get(user=user, repo_name=repo_name, repo_owner=repo_owner).delete()
```

Comment:
This logic does need to be put into manager.py to be merged. `def repo_add_collaborator` will also need to be updated. See `core.db.manager.list_create_card` and  `browser.views.card_create` to see how it's done. The reason are that
- we're using manager.py as a jumping off point for the REST api
- manager.py needs to check that the user has permissions on the repo_base and repo, to verify that they can add/delete collaborators.